src/test/integration/java/com/commercetools/pspadapter/paymentHandler/impl/PaymentHandlerProviderImplIT.java

```diff
@@ -201,12 +210,56 @@ public void shouldCreateChargeTransaction() {
         assertThat(chargeTxn.getState()).isEqualTo(SUCCESS);
     }
 
+    @Test
+    public void whenCtpPaymentIsAlreadyMarkedAsSuccess_shoudNotCallCreateTransactionInPaypalPlus() {
+
```

Comment:
Remove this line

Reply:
c80e2bc063492bda8fa0db75e748dae20b5e987f